logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/mlsmigration/MLSMigrator.kt

```diff
@@ -135,16 +137,10 @@ internal class MLSMigratorImpl(
             .flatMap { protocolInfo ->
                 when (protocolInfo) {
                     is Conversation.ProtocolInfo.Mixed -> {
-                        mlsConversationRepository.establishMLSGroup(protocolInfo.groupId, emptyList())
-                            .flatMap {
-                                conversationRepository.getConversationMembers(conversationId).flatMap { members ->
-                                    mlsConversationRepository.addMemberToMLSGroup(
-                                        protocolInfo.groupId,
-                                        members,
-                                        protocolInfo.cipherSuite
-                                    )
-                                }
-                            }
+                        conversationRepository.getConversationMembers(conversationId).flatMap { members ->
```

Comment:
what if conversation has no members, shouldn't we migrate it too. Not sure if it's possible, but still.

Reply:
I think the list will be empty and we continue with the migration!

Reply:
see the function's docs:
`
/**
     * Fetches a list of all members' IDs or a given conversation including self user
     */
`